src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/IconUrlToImageCacheConverter.cs

```diff
@@ -54,8 +61,62 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             var iconBitmapImage = new BitmapImage();
             iconBitmapImage.BeginInit();
-            iconBitmapImage.UriSource = iconUrl;
 
+            BitmapSource imageResult;
+
+            if (values.Length > 1
```

Comment:
```suggestion
            if (values.Length == 2
```

You never use `values[2]` or higher, so this block doesn't really support `values.Length > 2`

Reply:
I was trying to be flexible. I will change the code to expect exactly two values.